app/assets/stylesheets/global/_layout.scss

```diff
@@ -57,7 +61,7 @@ header {
     display: inline-block;
     width: 190px;
     height: 31px;
-    margin-right: 10px;
+    margin-#{$right}: 10px;
```

Comment:
You know what would be cool (for a separate issue, not in this PR) would be a git hook on this repo that flags any non-RTL-friendly CSS rules on commit. So you'd match for anything like `-(right|left):` or `: (right|left);`. Otherwise stuff will keep creeping in... You'd need to test it on the existing stylesheets and fix any existing issues so we start out with a clean set... Thoughts?

Reply:
You could even do a blog post about this. I bet others could use it...